Build.md

```diff
@@ -2,100 +2,198 @@
+
+You'll also need to install these dependencies using Visual Studio Installer (included with main installer):
+
+- Workloads 
+  - **Desktop development with C++**
+- Individual components
+  - **C++ Windows XP Support for VS 2017 (v141) tools**
+  - **MSVC v141 - VS 2017 C++ x64/x86 build tools (v14.16)**
+  - **Windows Universal CRT SDK**
+  
+#### Visual Studio 2019
+
+Download [VS 2019](https://visualstudio.microsoft.com/vs/) and install it.
+
+You'll also need to install these dependencies using Visual Studio Installer (included with main installer):
+
+- Workloads
+  - **Desktop development with C++**
+  
+#### Project dependencies
+
+- Download and install [Qt](https://www.qt.io/download) for MSVC 2017 32-bit
+  - **Important**: You have to create a personal account
+  - Minimum required version is `5.9`
+- Download and install latest version of [CMake](http://www.cmake.org) for Windows
+  - Make sure to add `cmake` as [global or user environment variable](https://support.shotgunsoftware.com/hc/en-us/articles/114094235653-Setting-global-environment-variables-on-Windows)
+- Download and install latest version of [pandoc](http://www.pandoc.org)
+
+### Project configuration with cmake
+
+Create a subdirectory in TrenchBroom directory called `build`.
+
+#### Visual Studio 2017
+
+Open a command prompt and change directory to `build`:
+
+```bash
+cd <path/to/TrenchBroom>/build
+```
+
+Then, execute this command to configure the project:
+
+```bash
+cmake .. -T v141 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH="C:\Qt\5.13.0\msvc2017"
+```
+
+> **Notes**:
+> - Specifying `CMAKE_BUILD_TYPE` is required when running cmake; you can't change between `Release` and `Debug` from Visual Studio.
+> - Make sure to specify the correct Qt as `CMAKE_PREFIX_PATH` value.
+
+#### Visual Studio 2019
+
+Open a command prompt and change directory to `build`:
+
+```bash
+cd <path/to/TrenchBroom>/build
+```
+
+Then, execute this command to configure the project:
+
+```bash
+cmake .. -G "Visual Studio 16 2019" -A Win32 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH="C:\Qt\5.13.0\msvc2017"
+```
+
+> **Notes**:
+> - Specifying `CMAKE_BUILD_TYPE` is required when running cmake; you can't change between `Release` and `Debug` from Visual Studio.
+> - Make sure to specify the correct Qt as `CMAKE_PREFIX_PATH` value.
+
+### Startup project with Visual Studio
+
+Open `TrenchBroom.sln` located into `build` directory.
+
+In the left sidebar, right-click on the header called `Solution 'TrenchBroom' (X of x projects)` and select the contextual option `Set Startup Project...`.
+
+A new window is opened. Set `TrenchBroom` as `Single startup project` and confirm clicking on `OK` button.
+
+### Build and debug TrenchBroom
+
+To build and debug TrenchBroom, press `F5` key. After a success compilation process, TrenchBroom should automatically be opened in `Debug` mode.
+
+---
 
 ## Linux
-### Dependencies
-TrenchBroom depends on:
```

Comment:
It might be better to keep this list because it the only place we state that our minimum GCC version is 7.
It's meant to be helpful for people on non-Debian distros so they don't have to look at the apt-get packages below (not all of which are really requirements)